### DISCUSSION TITLE
检测样本标注。 有的文字一小部分在矩形外，要不要放到框里面？ 在空格处分开，好一点？

### DISCUSSION THREAD

**Discussion by nissansz** [PFCCLab/PPOCRLabel#126] [Q&A]
![image](https://github.com/user-attachments/assets/700311bc-5c66-4819-9cfa-d78d66644a50)

**Comment by GreatV** [comment]
根据您的问题，关于检测样本标注时文字部分在矩形框外的处理，以及空格处是否分开标注，有以下建议：

1. **针对文字一小部分在矩形框外的情况**：
   - 在标注时，应尽量将文字完全包含在矩形框内。这是因为训练模型时，标注框的精确性会直接影响检测和识别的效果。如果文字的一部分在框外，可能会导致模型误判或漏检。
   - 您可以适当扩大矩形框的范围，将文字的完整部分纳入框内。但需注意，框的范围不宜过大，应尽量贴合文字的实际位置和大小。

2. **关于空格处分开标注是否更好**：
   - 是否在空格处分开标注取决于具体的训练目标。如果您的模型需要对每个单独的词进行识别，建议在空格处分开标注每个词；如果目标是识别整行或整段文字，则可以将空格包含在一个整体框内。
   - 从标注工具的使用来看，参考类似 [Issue #92](https://github.com/PFCCLab/PPOCRLabel/issues/92) 中提到的规则和顺序排列，空格的处理需要根据特定任务需求和标注规范来决定。

3. **其他相关优化建议**：
   - 如果您觉得手动调整框顺序复杂，可参考 [Issue #103](https://github.com/PFCCLab/PPOCRLabel/issues/103) 中提到的建议，即使用工具自动按照从上到下、从左到右调整顺序，减少人工操作的工作量。
   - 若标注工具或流程有其他优化需求，可以参考 [Issue #102](https://github.com/PFCCLab/PPOCRLabel/issues/102) 中提到的一些功能改进建议，例如保存和处理的便捷性。

总结来说：
- 文字尽量完全包含在矩形框内。
- 空格是否分开标注需要根据任务目标来决定。
- 合理利用工具的自动化功能以提高效率。

如需进一步讨论或有其他相关问题，可以在上述提到的相关 Issue 页面中进行查看或留言。

---
*Response generated by [feifei-bot](https://github.com/GreatV/feifei) | chatgpt-4o-latest*

**Comment by nissansz** [comment]
单张图片上多少行文字比较合适？500张就可以用来微调？
